tests/docker_cluster.py

```diff
@@ -61,14 +61,20 @@ def __init__(self, master_host, slave_hosts,
         # container mount point call get_local_mount_dir()
         self.local_mount_dir = local_mount_dir
         self.mount_dir = docker_mount_dir
+        self._client = None
 
+        self._DOCKER_START_TIMEOUT = 30
+        DockerCluster.__check_if_docker_exists()
+
+    @property
+    def client(self):
+        if self._client:
+            self._client.close()
```

Comment:
if the issue is multiple clients streaming, doesn't closing the client here cause problems?